src/components/Popup/Popup.tsx

```diff
@@ -126,39 +126,36 @@ export default class Popup extends AutoControlledComponent<Extendable<PopupProps
     trigger: PropTypes.any,
   }
 
-  public static defaultProps: PopupProps = {
+  static defaultProps: PopupProps = {
     accessibility: popupBehavior,
     align: 'start',
     position: 'above',
   }
 
-  public static autoControlledProps = ['open', 'target']
-
-  private static isBrowserContext = isBrowser()
-
-  private outsideClickSubscription = EventStack.noSubscription
+  static autoControlledProps = ['open', 'target']
 
-  private triggerDomElement = null
-  private popupDomElement = null
+  outsideClickSubscription = EventStack.noSubscription
+  triggerDomElement = null
+  popupDomElement = null
 
   protected actionHandlers: AccessibilityActionHandlers = {
     toggle: e => {
-      this.trySetOpen(!this.state.open, e, true)
+      this.trySetOpen(!this.state.open, e)
     },
     closeAndFocusTrigger: e => {
       this.closeAndFocusTrigger(e)
       e.stopPropagation()
     },
   }
 
-  private closeAndFocusTrigger = e => {
```

Comment:
this naming is used in lots of files (`private` vs `public`), and it doesn't provide any harm. More than that, occasionally it was helpful to folks to differentiate React methods from custom ones of the component.

I don't think that we should change this in that PR, especially if we are talking about single component file - it should be done either for all or not done at all (we've already had related discussions and agreed to proceed with explicit names for visibility scopes), but in either case it shouldn't be a matter of change for this PR

Reply:
OK, public/private make no sense in a React class.  I will address it separately.